packages/client/src/views/SysAdmin/Config/Integrations/Integrations.tsx

```diff
@@ -24,6 +54,72 @@ export const statuses = {
 
 export function IntegrationList() {
   const intl = useIntl()
+  const [showModal, setToggleModal] = React.useState<boolean>(false)
+  const [createClientInfo, setCreateClientInfo] = React.useState<boolean>(true)
+  const [generateClientInfo, setGenerateClientInfo] =
+    React.useState<boolean>(false)
+  const [clientName, setClientName] = React.useState<string>(EMPTY_STRING)
+  const [clientType, setClientType] = React.useState<string>(EMPTY_STRING)
+  const [selectedTab, setSelectedTab] = React.useState<string>(
+    WebhookOption.birth
+  )
+  const [selectedItems, setSelectedItems] = useState(['registration-details'])
+  const [selectedItemsNoPII, setSelectedItemsNoPII] = useState([
+    'registration-details-noPII'
+  ])
+  const [NoPII, setSelected] = React.useState(false)
+
+  const toggleModal = () => {
+    setToggleModal((prev: any) => !prev)
+  }
+
+  const changeModalInfo = () => {
+    setCreateClientInfo(!createClientInfo)
+    setGenerateClientInfo(!generateClientInfo)
+  }
+
+  const AlertSyled = styled(Alert)`
+    --color: rgb(56, 162, 157);
+    margin-top: 30px;
+    min-height: 5rem;
+  `
+  const TextInputStyled = styled(TextInput)`
+    width: 100%;
+  `
+
+  const SelectOptionStyled = styled(Select)`
+    width: 100%;
+  `
+  const DivSection = styled.div`
+    margin-top: 20px;
+  `
+
+  const Border = styled.div`
+    border-top: 1px solid rgb(238, 238, 238);
```

Comment:
[`<Divider />`](https://ui-kit.farajaland-qa.opencrvs.org/?path=/docs/layout-divider--template) -component might be what you are looking for